generators/app/templates/src/components/AuthNavBar/children/NavbarProfile/nav-bar-profile.component.js

```diff
@@ -46,35 +46,37 @@ class NavBarProfile extends Component {
     }
   };
   render() {
-    const { img } = this.props;
+    const { t, img } = this.props;
     const { imageLoaded } = this.state;
 
     const profileOpts = [
       {
-      label: "Profile",
-      onClick: this.profileRedirect
+        label: "Profile",
```

Comment:
This label should be from i18n similar to logout

Reply:
done